### DISCUSSION TITLE
what to save ?

### DISCUSSION THREAD

**Discussion by keneichi** [dani-garcia/vaultwarden#3328] [Q&A] [answered]
Hi guys, 

until very recently I was using BitWarden_RS with SQLite. Also I had no problem backing up my BitWareden_RS data to an external server.

Yesterday I switched to VaultWarden installed in docker.
My question is therefore, being an almost complete neophyte in docker, what should I save on this new installation?
is there a database to back up? If yes which one ? Is it SQLite?

Currently I use a script based on rsync which backs up the entire virtual machine (except dev, proc, sys, run) and therefore also backs up the docket container. But this method may not be suitable, if there is a mysql type database.

Thanks for help :)

Vincent

**Comment by BlackDex** [comment, marked as answer]
Vaultwarden is not different from Bitwarden_RS, it was just a rename. So all is technically still the same.

I don't know which database type you use, but the default is SQLite3, unless you configured otherwise.
Further, just backup the data folder should be enough, unless you configured other folders your self, then you should backup those of course.

**Reply by keneichi** [reply]
Well... the install is quite différent... I install ButWarden_RS not in docker. At this thime I could choose between SQLite, MySQL or PostgreSQL (I guess). 
Today, I just ran this two commands 
`docker pull vaultwarden/server:latest`
`docker run -d --name vaultwarden -v /vw-data/:/data/ -p 80:80 vaultwarden/server:latest`
And it work :)
And it is cool. But as I said before I am a newbie in docker. So I don't know how to see "how it work"... so i don't know what i have to backup :)

**Reply by keneichi** [reply]
ahaha
You have edited your message while i was respondind to you... and you answered to my question 
I didn't made any change in docker's contenair so SQLite is used. 
My script should still work

So thank you :)

**Reply by BlackDex** [reply]
Your welcome.

It's true that before there were separate versions per db type. But that was merged, even with BitwardenRS already.
